### PR TITLE
[SWE-58] Clean Codebase

### PR DESCRIPTION
**Pull request by sammyoto**
Added prettier and made it run on pre-commit with husky. Added an npm run style:all command and ran it on the codebase, that's why there are so many changes.

### CODE REVIEW

**Comment by sammyoto** [comment]
Some resources I used:
[Husky, Prettier, Eslint tutorial](https://tapajyoti-bose.medium.com/automatically-format-your-code-on-git-commit-using-husky-eslint-prettier-in-9-minutes-4dd6233f32b1)
[Configuring Nextjs with Eslint and Prettier](https://nextjs.org/docs/app/building-your-application/configuring/eslint#prettier)

**Review by datderek — CHANGES_REQUESTED** [review]
Looks like it ran correctly!

But is there a `.prettierignore` file? It looks like it is running on a lot of files it doesn't have to: Amplify config files, Tailwind config files, README.md, etc.

Nit: Also the default styling looks good, but maybe a `.prettierrc.json` config file should be included so we know what the style standards are.

**Review by datderek — APPROVED** [review]
Look good! Thanks for adding the style and config files.

File: README.md
Comment (jackfales) [inline README.md:1]: Add the style command to this file

**Review by jackfales — APPROVED** [review]
Nice

**Pull request merged**